src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -323,121 +307,168 @@
     animation: sf-blink .5s ease-in-out infinite;
 }
 @-webkit-keyframes sf-blink {
-    0% { color: black; }
-    50% { color: #bbb; }
-    100% { color: black; }
+    0% { background: #222; }
+    50% { background: #444; }
+    100% { background: #222; }
 }
 @-moz-keyframes sf-blink {
-    0% { color: black; }
-    50% { color: #bbb; }
-    100% { color: black; }
+    0% { background: #222; }
+    50% { background: #444; }
+    100% { background: #222; }
 }
 @-o-keyframes sf-blink {
```

Comment:
``-o-keyframes`` is useless. It was never supported prefixed before Opera switched their rendering engine